projects/software-assessment.manifest.yml

```diff
@@ -0,0 +1,35 @@
+slug: software-assessment
+name: Software Assessment
+description: Software Assessment Description
```

Comment:
A bundle of Scriptbee plugins and a linker for all Scriptbee plugins related to Software Assessment developed in dxworks